docs/_docs/extensions-and-integrations/spring/spring-data.adoc

```diff
@@ -152,17 +184,37 @@ public class SpringAppCfg {
     }
 }
 ----
+tab:Ignite thin client connection configuration[]
+[source,java]
+----
+@Configuration
+@EnableIgniteRepositories
+public class SpringAppCfg {
+    /**
+     * Creating Apache Ignite thin client instance bean. A bean will be passed to the IgniteRepositoryFactoryBean to
+     * connect to the Ignite cluster and perform cache operations.
+     */
+    @Bean
+    public IgniteClient igniteInstance() {
+        return Ignition.startClient(new ClientConfiguration().setAddresses("127.0.0.1:10800");;
+    }
+}
+----
 --
 
-The configuration has to instantiate Apache Ignite bean (node) that will be passed to `IgniteRepositoryFactoryBean`
-and will be used by all the Apache Ignite repositories in order to connect to the cluster.
+The configuration has to instantiate Apache Ignite bean (node) or Apache Ignite thin client bean that will be passed to
+`IgniteRepositoryFactoryBean` and will be used by all the Apache Ignite repositories in order to connect to the cluster.
 
 In the example above, the bean is initialized directly by the application and is named `igniteInstance`.
 Alternatively, the following beans can be registered in your configuration and an Apache Ignite node will be started automatically:
 
 * `IgniteConfiguration` object named as `igniteCfg` bean.
 * A path to Apache Ignite's Spring XML configuration named `igniteSpringCfgPath`.
 
+In the case of connecting to the cluster via Apache Ignite thin client, you can alternatively register the
+`ClientConfiguration` bean named `igniteCfg` so that the Apache Ignite thin client instance will be started automatically
+ by Apache Ignite Spring Data integration.
```

Comment:
```suggestion
In the case of connecting to the cluster via Apache Ignite thin client, you can alternatively register the `ClientConfiguration` bean named `igniteCfg` so that the Apache Ignite thin client instance is started automatically by the Apache Ignite Spring Data integration.
```